chain/pool/txpool.go

```diff
@@ -22,6 +22,7 @@ type TxnPool struct {
 	TxLists              sync.Map // NonceSortedTxs instance to store user's account.
 	TxMap                sync.Map
 	TxShortHashMap       sync.Map
+	NanoPayTxs           sync.Map // tx with nano pay type.
```

Comment:
How can miner package nanoPay transaction in mining.go? this map is not used in GetAllTransactionLists.

Reply:
I think you are right, GetAllTransactionLists() needs to be modified also I believe, though I'm not sure what would be a good order, like regular txn first or nanopay first

Reply:
I believe it got lost after one of rebases, I'll bring it back.

Reply:
I believe it got lost after one of rebases, I'll bring it back.

Reply:
fixed